src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -144,6 +145,19 @@ def get_commits(self, project, repo, fromHash, toHash):
             params={"since": fromHash, "until": toHash, "merges": "exclude"},
         )
 
+    def get_last_commits(self, project, repo, limit=10):
+        return self.get(
+            BitbucketServerAPIPath.repository_commits.format(project=project, repo=repo),
+            auth=self.get_auth(),
+            params={"merges": "exclude", "limit": limit},
+        )
+
+    def get_changelist(self, project, repo, commit):
```

Comment:
@pierredup I noticed none of the other integrations have this function. What's different about Bitbucket Server that requires it?

Reply:
Bitbucket has a similar function (just using a different name)
https://github.com/getsentry/sentry/blob/953683e2e0a42ce17a9b827cf61cf9bcbb9b0057/src/sentry/integrations/bitbucket/client.py#L122-L128

I'll update this to match the name used in the Bitbucket integration